### PR TITLE
4.0 - New Assignments and Behavior Name Change

### PR DESCRIPTION
**Pull request by 01Parzival10**
Added handling for new Assignments to Web2DFD and DFD2Web Converter
Adjusted Web2DFD and DFD2Web Converter to new Assignment semantic
Adjusted Behavior Naming where necessary
New tests for new Assignments

!!Forth Merge of 4.0 - Do DataFlowAnalysis first!!

### CODE REVIEW

**Comment by 01Parzival10** [comment]
Part of DataFlowAnalysis/DataFlowAnalysis#175

**Review by Nicolas-Boltz — DISMISSED** [review]
Same as with the 3. PR. Please re-run the build action of this PR once more after the previous PRs have been merged, just to be sure.

File: bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/DataFlowDiagramConverter.java [hunk 1]
```diff
@@ -263,27 +265,44 @@ private void createNodes(DataFlowDiagram dataFlowDiagram, List<Child> children,
...
     }
+    
+    private HashMap<Flow, String> createControlFlowNameMap(DataFlowDiagram dataFlowDiagram) {
+    	var controlFlowNameMap = new HashMap<Flow, String>();
+    	dataFlowDiagram.getNodes().forEach(node -> {
+    		var controlFlows = dataFlowDiagram.getFlows().stream()
+	    		.filter(flow -> flow.getDestinationNode().equals(node))
+	    		.filter(flow -> flow.getEntityName().equals("")).toList();
+    		
+    		String controlFlowName = "" + CONTROL_FLOW_NAME;
```
Comment (uuqjz) [inline bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/DataFlowDiagramConverter.java]: Didnt you do this code in a previous PR

Comment (01Parzival10) [reply bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/DataFlowDiagramConverter.java]: Yes, to run some tests i also copied it over to  this branch

**Review by uuqjz — APPROVED** [review]
LGTMN

**Pull request merged**